tests/test_003-config.py

```diff
@@ -186,8 +186,15 @@ def test_load_config():
     t.eq(app.cfg.workers, 3)
     t.eq(app.cfg.proc_name, "fooey")
 
+def test_load_config_explicit_file():
+    with AltArgs(["prog_name", "-c", "file:%s" % cfg_file()]):
+        app = NoConfigApp()
+    t.eq(app.cfg.bind, ["unix:/tmp/bar/baz"])
+    t.eq(app.cfg.workers, 3)
+    t.eq(app.cfg.proc_name, "fooey")
+
 def test_load_config_module():
-    with AltArgs(["prog_name", "-c", cfg_module()]):
+    with AltArgs(["prog_name", "-c", "python:%s" % cfg_module()]):
```

Comment:
From my understanding of your [comment](https://github.com/benoitc/gunicorn/issues/836#issuecomment-50957348), we do want to break backward compatibility here in order to separate the two cases. What do you think?